.env.example

```
@@ -15,6 +15,15 @@ PROJECT_NAME="Portabase"
 PROJECT_URL=http://localhost:8887
 PROJECT_SECRET=
 
+AUTH_DEFAULT_USER_NAME="Portabase Admin"
+AUTH_DEFAULT_USER=user@example.com
+# Password must contain at least 8 characters
+# Password must contain at least 1 number
+# Password must contain at least 1 lowercase letter
+# Password must contain at least 1 uppercase letter
+# Password must contain at least 1 special character
+AUTH_DEFAULT_PASSWORD=testPASS123456
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The AUTH_DEFAULT_PASSWORD key should go before the AUTH_DEFAULT_USER key

(UnorderedKey)